front-end/src/ui-components/Address.tsx

```diff
@@ -55,25 +59,29 @@ const Address = ({ address, className, displayInline, extensionName, popupConten
 			<Identicon
 				className='image identicon'
 				value={address}
-				size={displayInline ? 16 : 32}
+				size={displayInline ? 20 : 32}
 				theme={'polkadot'}
 			/>
 			<div className='content'>
 				{displayInline
 					// When inline disregard the extension name.
 					? popupContent
-						? <Popup
-							trigger={
-								<div className={'header inline'}>
-									{mainDisplay || shortenAddress(address)}
-									{sub && <span className='sub'>/{sub}</span>}
-								</div>
-							}
-							content={popupContent}
-							hoverable={true}
-							position='top center'
-						/>
+						? <>
+							{identity && mainDisplay && <IdentityBadge identity={identity}/>}
```

Comment:
I know apps is showing the IdentityBadge on the left, but having it right feels more natural to me (probably because of Twitter and others having it on the right)

Reply:
I tend to agree, but following the Polkadot guidelines.